Sim_To_DuT_Interface/DuT_Connectors/CANConnector/CANConnectorCodec.h

```diff
@@ -0,0 +1,70 @@
+/*******************************************************************************
```

Comment:
Bitte In diesem Projekt einheitlichen Header wie z.B. in der main.cpp

Reply:
Kann ich machen aber ich würde mMn. einen Abschnitt hinzufügen der den Sinn der Klasse kurz erkärt.
Generell: Wenn man etwas als Standard definiert sollte man dies mit allen Teilnehmern frühzeitig und transparent kommunizieren.